src/main.js

```diff
@@ -1,84 +1,73 @@
-// Private Methods
-function extendConfig(defaults, options) {
-    let prop, extended = {};
-
-    for (prop in defaults) {
-        if (Object.prototype.hasOwnProperty.call(defaults, prop)) {
-            extended[prop] = defaults[prop];
-        }
-    }
-
-    for (prop in options) {
-        if (Object.prototype.hasOwnProperty.call(defaults, prop)) {
-            extended[prop] = options[prop];
-        }
-    }
-
-    return extended;
-};
-
 export default class Concertina {
-    constructor(el, options) {
+    constructor(el, customOptions) {
         // Config
         this.element = el; // TODO: Get this to work with multiple elements - document.querySelectorAll('blah');
-        this.defaults = {
+        this.options = {
             'blockClass': '.Concertina',
             'tabClass': '.Concertina-tab',
             'tabActiveClass': '.is-selected',
             'panelClass': '.Concertina-panel',
             'panelActiveClass': '.is-visible'
         };
-        this.options = extendConfig.bind(this)(this.defaults, options);
-        this.tabs = this.element.querySelectorAll(this.options.tabClass);
-        this.panels = this.element.querySelectorAll(this.options.panelClass);
+
+        for (let prop in customOptions) {
+            this.options[prop] = customOptions[prop];
+        }
+
+        this.tabs = [...this.element.querySelectorAll(this.options.tabClass)];
+        this.panels = [...this.element.querySelectorAll(this.options.panelClass)];
```

Comment:
I use ES6 spread operators here to convert the NodeList object into a regular Array. (which has more useful methods which get used later)